### PR TITLE
mandown 1.0.1 (new formula)

### PR DESCRIPTION
**Pull request by DiegoMagdaleno**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----
Man down is like a manpage reader but for Markdown files.

### CODE REVIEW

File: Formula/mandown.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    system "make", "install", "PREFIX=#{prefix}"
+  end
+
+  test do
+    (testpath/"test.md").write <<~EOS
+      #Hi from readme file!
+    EOS
+    system "#{bin}/mdn", "-f", "test.md", "-o", "test"
+  end
+end
```
Comment (chenrui333) [inline Formula/mandown.rb:20]:
```
== Formula/mandown.rb ==
C: 20:  4: Final newline missing.
```

**Comment by chenrui333** [comment]
@DiegoMagdaleno thanks for the PR, can you also update the commit message as `mandown 1.0.1 (new formula)`, thanks!

**Comment by DiegoMagdaleno** [comment]
Sure thing @chenrui333

**Comment by DiegoMagdaleno** [comment]
@chenrui333 Name of the commit changed, thanks for the feedback!

**Comment by DiegoMagdaleno** [comment]
@chenrui333 Hi! I need some assitance, well the test failed in macOS 10.14 due to the outdated include files, so I opened a pull request for the developer: 

https://github.com/Titor8115/mandown/pull/24

Since I think I should use inreplace for this specific case.

And the test on macOS 10.13 tells me I should do "uses_from_macos" but those are outdated, what should I do? it works on my machine on catalina, I would appreciate feedback.

**Comment by DiegoMagdaleno** [comment]
Apparently it also failed in Catalina, but it worked on my machine, any guidance? Thanks, greetings.

File: Formula/mandown.rb [hunk 2]
```diff
@@ -0,0 +1,20 @@
+class Mandown < Formula
+  desc "Man-page inspired Markdown viewer"
+  homepage "https://github.com/Titor8115/mandown"
+  url "https://github.com/Titor8115/mandown/archive/v1.0.1.tar.gz"
+  sha256 "b014a44b27f921c12505ba4d8dba15487ca2b442764da645cd6d0fd607ef068c"
+
+  depends_on "libxml2"
+  depends_on "ncurses"
```
Comment (chenrui333) [inline Formula/mandown.rb:8]:
```suggestion
  uses_from_macos "libxml2"
  uses_from_macos "ncurses"
```

**Comment by chenrui333** [comment]
Applied a fix, seems working for me (Mojave machine)

**Comment by chenrui333** [comment]
Not sure how this error message got into the build.
```
==> FAILED
== Formula/pandoc-include-code.rb ==
C: 13:  3: license (line 13) should be put before bottle (line 6)
```

**Comment by DiegoMagdaleno** [comment]
@chenrui333 Thanks a lot! I now have to wait for the tests, however, the brew test on macOS fails due to an external formulae.

**Comment by chenrui333** [comment]
@DiegoMagdaleno, can you squash the three commits into one and forced update the PR? Thanks!

**Comment by DiegoMagdaleno** [comment]
Sure thing!

**Comment by DiegoMagdaleno** [comment]
@chenrui333 Commits squashed.

**Comment by chenrui333** [comment]
> @chenrui333 Commits squashed.

Thanks!

**Review by chenrui333 — DISMISSED** [review]
lgtm, only failed on the license audit check which is expected.

**Comment by DiegoMagdaleno** [comment]
Nice!, Thanks!

**Comment by chenrui333** [comment]
@DiegoMagdaleno it would great that you can also add this diff. 

```
index f4285916a16..fba0fbb871c 100644
--- a/Formula/mandown.rb
+++ b/Formula/mandown.rb
@@ -15,6 +15,13 @@ class Mandown < Formula
     (testpath/"test.md").write <<~EOS
       #Hi from readme file!
     EOS
+
+    expected_output = <<~EOS
+      <title >test.md(7)</title>
+      <h1>Hi from readme file!</h1>
+    EOS
+
     system "#{bin}/mdn", "-f", "test.md", "-o", "test"
+    assert_equal expected_output, File.read("test")
   end
 end
```

Thanks!

**Comment by DiegoMagdaleno** [comment]
@chenrui333 Requested changes are merged and squashed.

**Review by chenrui333 — APPROVED** [review]

**Pull request closed without merging**